main.tf

```diff
@@ -64,7 +64,7 @@ echo "#Pull values from Parameter Store and save to profile"
 touch /home/tableau_srv/env_vars.sh
 echo "
 export TABLEAU_ENVIRONMENT=internal
-export TABLEAU_REPO_ENVIRONMENT=internal
+export TABLEAU_REPO_ENVIRONMENT="'internal'"
```

Comment:
That doesn't look necessary...

Reply:
The TABLEAU_REPO_ENVIRONMENT environment variable needs to be enclosed in single quotes in order to be interpreted as a string and not a column by the psql script.  I have tried many other alternative approaches.